src/lib/makeRequest.ts

```
@@ -1,15 +1,64 @@
+  method,
+  data,
+}: {
+  url: string;
+  method: "GET" | "POST" | "PUT" | "DELETE";
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  data: Record<string, any>;
+}): Promise<{
   status: IncomingMessage["statusCode"];
   headers: IncomingMessage["headers"];
-  body: string;
+  body: Record<string, unknown> | any[];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)